src/context/AllProvider.jsx

```diff
@@ -0,0 +1,19 @@
+import { ArchiveProvider } from './ArchiveContext/ArchiveContext';
+import { ThemeProvider, AuthProvider, NotesProvider } from './index';
+import { TrashProvider } from './TrashContext/TrashContext';
+
+const AllProvider = ({ children }) => {
+	return (
+		<AuthProvider>
+			<ThemeProvider>
+				<NotesProvider>
```

Comment:
Appreciation: A great way to club the providers together.
One suggestion: Theme does not require auth so it can be above Auth provider

Reply:
done